hip-lang-config.cmake.in

```diff
@@ -94,6 +94,15 @@ if (HSA_HEADER-NOTFOUND)
   message (FATAL_ERROR "HSA header not found! ROCM_PATH environment not set")
 endif()
 
+file(GLOB HIP_CLANGRT_LIB_SEARCH_PATHS "${CMAKE_HIP_COMPILER}/../lib/clang/*/lib/*")
```

Comment:
CMAKE_HIP_COMPILER is the compiler not the directory where compiler lives. There is also no guarantee with full path. Gosh. This file is so broken.